scripts/curl_install_pypi/install.py

```diff
@@ -41,27 +38,71 @@
     pass
 
 AZ_DISPATCH_TEMPLATE = """#!/usr/bin/env bash
-{install_dir}/{bin_dir_name}/python -m azure.cli "$@"
+{install_dir}/bin/python -m azure.cli "$@"
 """
 
-DEFAULT_INSTALL_DIR = os.path.join(os.path.sep, 'usr', 'local', 'az')
-DEFAULT_EXEC_DIR = os.path.join(os.path.sep, 'usr', 'local', 'bin')
+DEFAULT_INSTALL_DIR = os.path.expanduser(os.path.join('~', 'azure-cli'))
+DEFAULT_EXEC_DIR = os.path.expanduser(os.path.join('~', 'bin'))
 VIRTUALENV_VERSION = '15.0.0'
-BIN_DIR_NAME = 'Scripts' if platform.system() == 'Windows' else 'bin'
 EXECUTABLE_NAME = 'az'
 
-DISABLE_PROMPTS = os.environ.get('AZURE_CLI_DISABLE_PROMPTS')
+USER_BASH_RC = os.path.expanduser(os.path.join('~', '.bashrc'))
+USER_BASH_PROFILE = os.path.expanduser(os.path.join('~', '.bash_profile'))
+COMPLETION_FILENAME = 'az.completion'
+PYTHON_ARGCOMPLETE_CODE = """
+
+_python_argcomplete() {
+    local IFS='\v'
+    COMPREPLY=( $(IFS="$IFS"                   COMP_LINE="$COMP_LINE"                   COMP_POINT="$COMP_POINT"                   _ARGCOMPLETE_COMP_WORDBREAKS="$COMP_WORDBREAKS"                   _ARGCOMPLETE=1                   "$1" 8>&1 9>&2 1>/dev/null 2>/dev/null) )
+    if [[ $? != 0 ]]; then
+        unset COMPREPLY
+    fi
+}
+complete -o nospace -F _python_argcomplete "az"
+"""
+
+class CLIInstallError(Exception):
+    pass
+
+def print_status(msg=''):
+    print('-- '+msg)
+
+def prompt_input(msg):
+    return input('\n===> '+msg)
+
+def prompt_input_with_default(msg, default):
+    if default:
+        return prompt_input('{} (leave blank to use {}): '.format(msg, default)) or default
```

Comment:
Recommend adding single quotes around the default to separate it from the message:
`"{} (leave blank to use '{}': "`